src/styles/blocks/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $accent-color;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    display: block;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__review {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: center;
+
+    &-text {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $accent-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+    align-items: center;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-price {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $accent-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
+    background: $button-color;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
```

don't forget to create variables for all colors
```suggestion
    color: $c-white;
```